api/src/pcapi/routes/backoffice_v3/custom_reimbursement_rules.py

```diff
@@ -0,0 +1,104 @@
+from flask import flash
+from flask import render_template
+import sqlalchemy as sa
+
+from pcapi.core.finance import models as finance_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offers import models as offers_models
+from pcapi.core.permissions import models as perm_models
+from pcapi.utils.clean_accents import clean_accents
+
+from . import autocomplete
+from . import utils
+from .forms import custom_reimbursement_rule as custom_reimbursement_rule_forms
+
+
+custom_reimbursement_rules_blueprint = utils.child_backoffice_blueprint(
+    "reimbursement_rules",
+    __name__,
+    url_prefix="/reimbursement-rules",
+    permission=perm_models.Permissions.READ_REIMBURSEMENT_RULES,
+)
+
+
+def _get_custom_reimbursement_rules(
+    form: custom_reimbursement_rule_forms.GetCustomReimbursementRulesListForm,
+) -> list[finance_models.CustomReimbursementRule]:
+    base_query = (
+        finance_models.CustomReimbursementRule.query.outerjoin(offers_models.Offer)
+        .outerjoin(offerers_models.Venue)
+        .outerjoin(offerers_models.Offerer)
```

Comment:
Comme il y a 2 jointures possibles sur `Offerer` :
- `CustomReimbursementRule.offerer`
- `CustomReimbursementRule.offer.venue.offerer`

je pense qu'il serait plus sûr de préciser explicitement : 
```suggestion
        .outerjoin(offerers_models.Offerer, offerers_models.Venue.offerer)
```
Je ne saurais cependant dire comme ça, si sqlalchemy peut faire la jointure sur le "mauvais" tel que tu l'as écrit.

Reply:
A priori tout marche, mais si ça ne coûte rien d'ajouter ça...

Reply:
Le sujet « Spécifier les chemins de jointures dans sqlAlchemy » avait été évoqué par @Meewan  dans la dernière synchro backend, le 3 avril. Il a peut-être un avis plus argumenté que moi sur le sujet :)

Reply:
Si à l'avenir on rajoute une deuxieme relation entre les Offerers et les Venues sqlalchemy risque de prendre la mauvaise relation sur la jointure si on ne le lui précise pas. Il y a deux syntaxes pour éviter le probleme

`.outerjoin(offerers_models.Offerer, offerers_models.Venue.managingOfferer)`
`.outerjoin(offerers_models.Venue.managingOfferer)`

Même remarque pour les autres jointures.

ps: attention le lien est sur managingOfferer et pas sur offerer

ps2: pour des questions de lisibilité comme tu join deux fois sur la même table ce serait bien d'utiliser des aliased pour qu'on sache sur quel jointure tu travail https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.aliased